### ISSUE TITLE
evdi-dkms doesn't install with kernel 5.11

### ISSUE THREAD

**Issue by swoopla** [pop-os/linux#46]
**Distribution:**
NAME="Pop!_OS"
VERSION="20.10"
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.10"
VERSION_ID="20.10"
VERSION_CODENAME=groovy
UBUNTU_CODENAME=groovy

I have a problem with **evdi-dkms** package:
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up evdi-dkms (1.7.0+dfsg-1ubuntu1) ...
Removing old evdi-1.7.0+dfsg DKMS files...

------------------------------
Deleting module version: 1.7.0+dfsg
completely from the DKMS tree.
------------------------------
Done.
Loading new evdi-1.7.0+dfsg DKMS files...
Building for 5.11.0-7612-generic
Building initial module for 5.11.0-7612-generic
ERROR (dkms apport): kernel package linux-headers-5.11.0-7612-generic is not supported
Error! Bad return status for module build on kernel: 5.11.0-7612-generic (x86_64)
Consult /var/lib/dkms/evdi/1.7.0+dfsg/build/make.log for more information.
dpkg: error processing package evdi-dkms (--configure):
 installed evdi-dkms package post-installation script subprocess returned error exit status 10
Errors were encountered while processing:
 evdi-dkms
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Regards

[evdi-dkms_build-error--make.log](https://github.com/pop-os/linux-firmware/files/6257082/evdi-dkms_build-error--make.log)

**Comment by jmstriegel** [comment]
I'm not positive, but I have a suspicion that this might be related to #48 and you'll find that there is a missing /usr/src/linux-headers-5.11.0-7612-generic/include/generated/autoconf.h. For some reason the linux-headers package isn't properly installing this after the latest 5.11 kernel update, and it appears to affect several dkms packages.

If so, it's likely an issue with the linux-headers package and not the driver itself.

**Comment by jacobgkau** [comment]
https://github.com/DisplayLink/evdi/issues/275 indicates this should be fixed now, and 5.11 is no longer the current kernel; closing.
